stdlib/tkinter/__init__.pyi

```diff
@@ -384,8 +391,34 @@ class Misc:
     @overload
     def grid_bbox(self, column: int, row: int, col2: int, row2: int) -> Tuple[int, int, int, int] | None: ...
     bbox = grid_bbox
-    def grid_columnconfigure(self, index, cnf=..., **kw): ...  # TODO
-    def grid_rowconfigure(self, index, cnf=..., **kw): ...  # TODO
+    # The cnf parameter's default value is an empty dict. That's a perfectly
+    # valid instance of _GridIndexInfo, but MyPy declares that an empty dict
+    # literal must be dict[Any, Any] and is therefore incompatible.
+    #
+    # The return value is always _GridIndexInfo if no optional parameters are
+    # provided, and always None if one or more are provided. Unfortunately,
+    # type inference requires both cases have the same return type, so we must
+    # combine them.
+    def grid_columnconfigure(
+        self,
+        index: _GridIndex,
+        cnf: _GridIndexInfo = ...,
+        *,
+        minsize: _ScreenUnits = ...,
+        pad: _ScreenUnits = ...,
+        uniform: str | None = ...,
```

Comment:
I don't think passing in `_None` does the right thing. The options get passed to `Misc._options`, which ignores all `None` values.

Reply:
You're right! I can't find it actually mentioned in the documentation anywhere, but it seems like empty-string is the way to remove a row or column from a uniform-group. I'll change the type annotation to just `str`.